python/animl/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM socrata/python
+
+MAINTAINER Nick Kypreos <nick.kypreos@socrata.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python-numpy python-scipy
+
+# pip installations
+RUN pip install geopy==1.5.0
```

Comment:
I would very strongly recommend you condense the RUN instructions into as few as practical by combining them, because docker.  Creating an image for each one seems unwise and slow and bloaty.